### ISSUE TITLE
Formal RFC: Stolen & user created Terraria content

### ISSUE THREAD

**Issue by hakusaro** [Pryaxis/handbook#2]
Hiya. I'd like to formally propose under the new RFC process that we adopt the following policy regarding downloading/uploading/sharing of Terraria worlds & TShock plugins without permission:

* If you profit (monetarily or socially) from unauthorized sharing of Terraria content (like user generated worlds or plugins), we reserve the right to remove you from the community.
* If you admit to stealing people's worlds or plugins, you can be removed from the community.
* If we are presented with proof that you are currently operating stolen content, you can be removed from the community.

This is in response to the threat that, if we were to not unban Marcus, we would be faced with the release of a tool that has the capability of stealing Terraria worlds. Unbanning Marcus is a separate issue, but like I told Ijwu, I think that we're well within our rights to die on this cross.

In particular, I want to support a thriving community of user generated content. I don't want our community to turn into the battleground for stolen content, nor do I want it to become known that we implicitly support servers (through technical support or allowing them to communicate over our services) when we know they're operating stolen content.

----

This RFC is up for discussion and vote until Tuesday, August 22, 2017. Please give a firm approve or deny, or give more opinion. Obviously, my opinion is above, and my suggestion is approve.

In either approval or denial, I can write a summary and officialize it.

**Comment by Ijwu** [comment]
Approved without much further comment from me. The important thing is that all the points imply intent to steal or operate stolen content. Intent is likely to be the key factor in judgement.

**Comment by DaBomberNTG** [comment]
All points approved by me.

**Comment by hakusaro** [comment]
@Pryaxis/tshock please voice opinions in this RFC thread before the 22nd.

**Comment by mistzzt** [comment]
I approve policies listed here.

**Comment by QuiCM** [comment]
👍

**Comment by Celant** [comment]
👍

**Comment by Zaicon** [comment]
👍

**Comment by SignatureBeef** [comment]
seems fair to me, so have a thumbs up 👍

**Comment by cheahjs** [comment]
👍

**Comment by Commaster** [comment]
Once this gets accepted, don't forget to remove yourselves from the community for stealing my OfflineBans implementation and socially profiting from it. Cheers.

**Comment by bartico6** [comment]
👍
It's good to have spoken rules saved to official statements, as then the "relative common sense" factor is eliminated.

**Comment by tlworks** [comment]
I agree with @Commaster on this matter. Might as well delete the community as a whole and honestly, you may ban me for it but with rules like this it won't get any better. You won't even be able to post build images anymore because it could mean that these particular builds could be stolen from another person.

Then again, I don't think anyone can copyright blocks or builds. I could simply recreate the whole building without 'stealing' it and it would still be deemed wrong. Therefore I think this is a stupid thing to add to the Nazism that's already playing.

**Comment by hakusaro** [comment]
@Commaster I literally have no idea what you're saying. If you have a specific grievance, you can always contact us and resolve it. We (I) offered to lift your ban several times with no problems, you just had to ask -- and never did.

@tlworks it's all about intent. If you *intend to profit* from stolen content, then you're surely not going to be welcome here.

**Comment by Patrikkk-zz** [comment]
This is a very small community. Sure, cannot copyright builds. But people should have mutual respect towards each other, when they are specifically asked not to use the content others spent hours creating.. @tlworks

**Comment by Patrikkk-zz** [comment]
Also thumbs  up 👍

**Comment by ivanbiljan** [comment]
👍
